src/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+}
+
+.form-search {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.input {
+  line-height: 22px;
+  font-size: 16px;
+  color: #3d4e61;
+  width: 100%;
+  box-sizing: border-box;
```

Comment:
use box-sizing: border-box for * selector. This rule is only one that allowed to use for this selector